package.json

```diff
@@ -194,16 +195,17 @@
     "react-router-query-params": "^1.0.3",
     "react-select": "^3.1.0",
     "react-table": "^7.6.1",
-    "redux": "^3.6.0",
+    "redux": "^4.1.2",
     "redux-devtools-extension": "^2.13.8",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
     "shortid": "^2.2.16",
     "slugify": "^1.4.6",
+    "stream-browserify": "^3.0.0",
     "stubborn-queue": "^2.0.0",
     "url": "^0.11.0"
   },
   "optionalDependencies": {
-    "cypress": "8.6.0"
+    "cypress": "^8.7.0"
```

Comment:
Do we want the `^` on cypress? That will allow cypress to update to any 8.x.x version.

Reply:
Good call! Removed